sed/config/mpes_example_config.yaml

```diff
@@ -79,6 +73,25 @@ dataframe:
     kx: '1/A'
     ky: '1/A'
 
+  # dataframe channels and group names to read from the h5 files
+  channels:
+    # The timestamp
+    X:
+      format: per_electron
+      group_name: "/Stream_0"
+    Y:
+      format: per_electron
+      group_name: "/Stream_1"
+    t:
+      format: per_electron
+      group_name: "/Stream_2"
+    ADC:
+      format: per_electron
+      group_name: "/Stream_4"
+    sampleBias:
+      format: per_file
```

Comment:
is this a slow channel format? perhaps we should also change the per_train channels to per_file if it's the same

Reply:
Yes, these are stored once per file (about 10 Mevents). Currently, this performs very poorly.

Reply:
I think the filling operations are always slow so there would be a reduction in performance. 
Also, per_train channels aren't necessarily stored per_file so I guess we need that